install_files/securedrop-grsec/DEBIAN/postinst

```diff
@@ -15,14 +15,21 @@ set -x
 #          <conflicting-package> <version>
 # for details, see http://www.debian.org/doc/debian-policy/ or
 # the debian-policy package
+GRSEC_VERSION='4.14.188-grsec'
+
+# Sets default grub boot parameter to the kernel version specified
+# by $GRSEC_VERSION. The debian buster default kernel is 4.19, thus
+# supersedes this 4.14.x series grsecurity kernel at boot-time
+set_grub_default() {
+    GRUB_OPT="'Advanced options for Debian GNU/Linux>Debian GNU/Linux, with Linux $GRSEC_VERSION'"
```

Comment:
In the grub menu, does Ubuntu still set `Debian GNU/Linux>Debian GNU/Linux`? That is, I'd expect "Ubuntu", not "Debian". See in https://github.com/freedomofpress/securedrop/blob/f81e41539bc5e5ae8b9a443ead719513622553c8/install_files/ansible-base/roles/grsecurity/tasks/from_fpf_repo_install_grsec.yml#L61

where we're explicitly referring to the menu entry as `Advanced options for Ubuntu>`. It's possible the appropriate settings have changed between Xenial & Focal, but at the very least we should be setting the same in both places.